runtime/runtime-core/jvm/src/aws/smithy/kotlin/runtime/time/InstantJVM.kt

```diff
@@ -26,6 +26,7 @@ import kotlin.time.toKotlinDuration
 import java.time.Duration as jtDuration
 import java.time.Instant as jtInstant
 
+// FIXME Consider making this multiplatform (`common`) using kotlinx.datetime
```

Comment:
**Comment**: Early in the SDK design, we rejected **kotlinx-datetime** as a backing impl because of concerns about support and long-term stability. These concerns might be addressed once [**kotlinx-datetime** is promoted to Beta](https://youtrack.jetbrains.com/issue/KT-64578/Promote-kotlinx-datetime-to-Beta) or once [`Instant` and `Clock` are moved to stdlib](https://youtrack.jetbrains.com/issue/KT-72480/Move-Instant-and-Clock-from-kotlinx-datetime-to-stdlib).